packages/transformer-markdown/src/index.ts

```diff
@@ -48,6 +50,7 @@ const transformer: TransformerPlugin = (config: MarkdownTransformerConfig) => {
       },
     }),
     inspect: (input: EntryNode) => String(input),
+    extensions: ['.md', '.mdx', '.markdown'],
```

Comment:
Does this remove the ability to specify custom extensions?

Reply:
this would leave the custom specification of extensions up to the transformer config, which it doesn't look like the markdown transformer exposes.  after looking at this again, it looks like this module doesn't actually support mdx, so maybe we should default it to only .md with the ability to customize it